app/support/reservations/validations.rb

```diff
@@ -21,10 +21,22 @@ module Reservations::Validations
 
     validate :starts_before_ends
     validate :duration_is_interval
+    validate :category_is_valid
   end
 
   # Validation Methods
 
+  def category_is_valid
+    return if category.blank?
+    if self.class.const_defined?(:CATEGORIES)
+      unless self.class::CATEGORIES.include?(category)
+        errors.add(:category, "must be a defined category")
```

Comment:
If we don't use my other suggestion, can we use `errors.add(:category, :invalid)` or whatever the `inclusion` validator uses so it grabs from the locale?